narayana-release-process.sh

```diff
@@ -198,55 +106,23 @@ else
+# uploaded artifacts go straight live without the ability to close the repo at the end, so the install is done to verify that the build will work
+./build.sh clean install -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -Dorson.jar.location=$ORSON_PATH -Pcommunity
 if [[ $? != 0 ]]
 then
   echo 1>&2 Could not install narayana
   exit
 fi
+# It is important in the deploy step that if you are deploying to nexus you provide a reference to your settings file as the ./build.sh overrides the default settings file discovery of Maven. Please see https://github.com/jbosstm/narayana/wiki/Narayana-Release-Process for details of the settings.xml requirements
 ./build.sh clean deploy -Dmaven.repo.local=${PWD}/localm2repo -DskipTests -gs ~/.m2/settings.xml -Dorson.jar.location=$ORSON_PATH -Prelease,community -DskipNexusStagingDeployMojo=false
 if [[ $? != 0 ]]
 then
   echo 1>&2 Could not deploy narayana to nexus
   exit
 fi
-git archive -o ../../narayana-full-$CURRENT-src.zip $CURRENT
-ant -f build-release-pkgs.xml -Drsync.host=${RSYNC_HOST} -Drsync.enabled=${RSYNC_ENABLED} -Dawestruct.executable="awestruct" all
-if [[ $? != 0 ]]
-then
-  echo 1>&2 COULD NOT BUILD Narayana RELEASE PKGS
-  exit
-fi
-cd -
-
-# Building and pushing the lra coordinator docker image
-cd ~/tmp/narayana/$CURRENT/sources/jboss-dockerfiles/lra/lra-coordinator
-git checkout $CURRENT
-if [[ $? != 0 ]]
-then
-  echo 1>&2 jboss-dockerfiles: Tag $CURRENT did not exist
-  exit
-fi
-
-if [[ $QUAYIOOK != n* ]]
-then
-  echo "building, tagging and pushing podman images to quay.io ..."
-#read -p "Have you logged in (podman login quay.io), if not you will be prompted (y/n): " ok
-#if [[ $ok == y* ]]; then
-  # build the lra-coordinator image
-  podman build --tag lra-coordinator --build-arg NARAYANA_VERSION=${CURRENT} .
-  # tag it and push it
-  podman tag lra-coordinator quay.io/jbosstm/lra-coordinator:${CURRENT}
-  podman tag lra-coordinator quay.io/jbosstm/lra-coordinator:latest
-
-  # push the image
-  podman push quay.io/jbosstm/lra-coordinator:${CURRENT}
-  podman push quay.io/jbosstm/lra-coordinator:latest
-  # the image will appear in https://quay.io/repository/jbosstm/lra-coordinator?tab=tags
-#else
-#  echo "alternatively run the above commands manually"
-#fi
-fi
 
-# not sure why we need to look at CI at this point so commenting it out
-# xdg-open http://narayanaci1.eng.hst.ams2.redhat.com/ &
+# Post-release steps
+echo "narayana.io needs updating, please update the narayana.io repository (see https://github.com/jbosstm/narayana.io/blob/develop/README.md)"
+echo "Please open a PR to lra-coordinator-quarkus when the artifact is available on nexus."
+echo "Please raise a Jira and pull request to update WildFly to the released version of Narayana"
+echo "Please visit Narayana CI and check the quickstarts are working with the release and obtain performance numbers for the blog post"
```

Comment:
I think it might be nicer to see the CI results before doing the other things in this block, what do you think?